src/runtime/opencl/opencl_common.h

```diff
@@ -508,26 +531,11 @@ class OpenCLTimerNode : public TimerNode {
   Device dev_;
 
   void recreateCommandQueue() {
-    cl_command_queue_properties prop;
-
     if (!cl::OpenCLWorkspace::Global()->IsProfiling(dev_)) {
-      prop = CL_QUEUE_PROFILING_ENABLE;
+      cl::OpenCLWorkspace::Global()->EnableQueueProfiling(dev_, true);
```

Comment:
Probably it will be better to do in such way:
```c++
cl::OpenCLWorkspace::Global()->EnableQueueProfiling(dev_, !cl::OpenCLWorkspace::Global()->IsProfiling(dev_));
```